### ISSUE TITLE
Open Source Mycetoma (MycetOS) Meeting 23 NOV 2021

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#66]
### **Open Source Mycetoma (MycetOS) Meeting**  

Meeting Zoom link [here](https://uni-sydney.zoom.us/j/99281549497)

**Time:** UTC 10:00 - London 11AM - Rotterdam Noon - Sydney 9 PM - New York 6 AM (Effective till February 2022 Meeting, Daylight saving changes in March meeting, requiring time revision) 

**Chair:** USYD 

**Minute taker:** Erasmus 

_Recording_ can be found here  

_Minutes_ can found here  

**Link to previous meeting** #65  
____________________________________________
### 1. Outstanding items from previous meeting 

- [ ] Dmitrij and Hung to agree on common grounds for colour-coding visual summaries (threshold or criteria for green/amber/red/black). We agreed to keep the visual summary concise and free of clutters such as compounds IDs, potency values etc (reserving these for tabulation of data). This conversation should continue at GHI Structural Activity Relationship Analysis of biological evaluation For Fenarimol Analogues/Epichem compounds #51
- [ ]   Manuscript needs urgent updates of Chemistry data (Dmitrij ready, Hung to do) @dmitrij176 @fantasy121 #55 
- [ ]   Literature reading is recommended to see if our SAR agree with exisiting SAR somewhere else. (See Keenan’s SAR papers of fenarimol for Chagas, and other papers). Point of interest for lit reading: Azoles vs fungal cytochrome P450 (CYP51 protein) @dmitrij176 @fantasy121

> Potential starting points (Keenan's papers on Fenarimols vs Chagas, SAR, CYP51 etc.)
> DOI: 10.1021/jm2015809
> DOI: 10.1016/j.bmc.2013.01.050
> DOI: 10.1074/jbc.M113.497990
> DOI: 10.1021/jm401610c

### 2. Discussion points for this meeting  

- USYD update
- UCL update
- Erasmus update

### 3. Any other businesses?

Meeting adjourned.

**Comment by mattodd** [comment]
Hi there - sorry but I'm an apology for today. @wwjvdsande might be able to update about the ISNTD session this Thursday. https://dndi.org/events/2021/isntd-connect-whats-in-the-pipeline-for-mycetoma/ 
It'd be very good if the discussion could briefly turn to a good target or two for @AndyBishopSevenoaks @kym834 ?

**Comment by wwjvdsande** [comment]
@mattodd, sure I can update about the ISNTD session this thursday. Yesterday I had my tour through the platform. @fantasy121 I did do some analysis on the LogD and the survival in the larvae which I can show during the meeting.

**Comment by kym834** [comment]
## Minutes below (please add anything if I've missed it or got something wrong)

**Paper and SAR updates**

@dmitrij176 found a paper that looks at some of the SAR of fenarimols. Interesting points for us to consider
* 2,4-difluoro has previously found to be important for activity. We have found that similar to this compounds we have tested that are 2-chloro-4-fluoro have been better than others. 
* Hydroxy group at the core was also found to lead to good activity. Again we have seen something similar in our reserach.
- [ ]@dmitrij176  will add the DOI to the list of papers to read above. 

@fantasy121 has pull out 12 compounds from the EpiChem library that would fill some gaps in our SAR. We need to consider if it is worth asking for these or if it would be quicker for us to make them instead. 
Also we need to find out if we need to pay for these compounds. There was confusing around this.
- [ ] @fantasy121 add an image of the 12 compounds to this issue so we can decide how we want to progress.

There are some gaps in the SAR around stereochemistry. This may be a place we explore if we continue this scaffold.

@fantasy121 and @dmitrij176 now have a few images that show the SAR in different ways. Still unsure on which to use. @kym834 suggested that this will depend on the story we are telling and where we find interesting results. 

@wwjvdsande has done some more statistical analysis and with all the current data, we have lost the statistical significance we observed in the last paper between LogD and activity (p = 0.7). 

There have been some issue with formatting for the experimental and characterisation data using google docs. We have now moved to Dropbox so that we can use Microsoft Word docs (#55). @kym834 has created a folder for Open Source Mycetoma which can accessed by anyone with the [link](https://www.dropbox.com/sh/myca0ofv9vq0ezf/AABBqZTzd9JXyhRSG1qDtdHia?dl=0). Those @OpenSourceMycetoma/corecontrib will be added specifically as well. @wwjvdsande has already moved the paper draft over.
- [ ] @fantasy121 and @dmitrij176 to bring together their two doc and share in this folder. 
- [ ] @kym834 to share folder with all those in the @OpenSourceMycetoma/corecontrib and add a link to the folder to the wiki so those who wish to can access it.


**Erasmus Updates**

Larvae season is over. Have not finished all of the testing with 4 compounds still to be tested _in vivo_. Can still send compounds for _in vitro_ testing and if we have any promising hits they can be tested _in vivo_ in the next larvae season. Plan to start up again in mid-late April 2022.

@MA-Jjingyi will be screening a new box from MMV which has 5000 compounds in it. We should be able to get some new hits from this. @wwjvdsande suggested looking at hits with different MOA would be interesting as we could then start to think about combinations.

**UCL Update**

@dmitrij176 has started working on the ketoxime scaffold. We have tested a couple of these compounds be for and identified that the pyridine ring and the 2,4 di-substitution are important for activity. The first compounds will explore modifications of the oxygen in the oxime functional group and look at chain length and cyclic substituents.

**USyd Update**

@fantasy121  is looking to explore some new chemistry to build his PhD Thesis and is considering starting to work on alternative series. @kym834 is also looking to begin working on Series 5 now that we have school and university working on Series 2.

@kym834 has two yr 12 students who will be beginning a data analysis/correlation project using some of the data from the screening boxes we've looked at so far. They will be meeting tomorrow to get started. I will suggest some further and deeper analysis on the fenarimol series as well as exploring either Series 2 or Series 5.
- [ ] @kym834 to update on the progress and what they students decide they would like to do after the meeting.

(update not from the meeting: John at IMSA has start the synthesis of the Series 2 compounds see https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/7 for more info)

**AOB**

Kaitlin (from the University of Auckland) was able to have her students do some work on the SAR for series 2 in the two short weeks she got with them this year. She will update us and share the results from this. They have made some suggestion on what compounds we might consider making next based on their analysis.

**Comment by MFernflower** [comment]
@kym834 Sorry I could not attend - Is there a lab notebook link for the group making new S2 molecules?

**Comment by PaulMcKeatingSevenoaks** [comment]
We are planning to share the webinar today with our students, as it's during their timetabled session for Breaking Good...

@mattodd @wwjvdsande @kym834 @andybishopsevenoaks

**Comment by kym834** [comment]
@PaulMcKeatingSevenoaks I've updated the top to include the link to the recording on YouTube

**Comment by wwjvdsande** [comment]
Dear all, here is the link on the mycetoma session
https://www.isntd.org/isntd-connect

**Comment by MFernflower** [comment]
@wwjvdsande Any recording of the session?

**Comment by MFernflower** [comment]
@dmitrij176 Would it be possible for you to post what antifungal-oximes you have made / plan to make?

**Comment by dmitrij176** [comment]
> @dmitrij176 Would it be possible for you to post what antifungal-oximes you have made / plan to make?

@MFernflower yes, I will create a new Github page on Ketoximes with the latest updates shortly.

**Comment by MFernflower** [comment]
@dmitrij176 <https://github.com/dmitrij176> maybe could spin them off into
a sub series on GitHub like series 1B or something like that?

The oximes likely hit the same enzyme as the fenarimols do

**Comment by MFernflower** [comment]
@wwjvdsande @dmitrij176 Quick Question - Has pyrifenox itself been tested vs madurella?

**Comment by MFernflower** [comment]
@dmitrij176 I think this may be worth following to see if we can get the ketoxime compounds logp down:

![image](https://user-images.githubusercontent.com/3164942/146286541-0f4632ea-e129-4171-90c2-c6701193e262.png)

**Comment by dmitrij176** [comment]
@MFernflower  That compound has already been made.

**Comment by dmitrij176** [comment]
> @wwjvdsande @dmitrij176 Quick Question - Has pyrifenox itself been tested vs madurella?

Pyrifenox has been tested in the last biological assay and gave very good results. If I remember correctly, the Minimum Inhibitory Concentration (MIC) was around 0.25 micromollar. Therefore, it will likely be used as a positive control in the future.

**Comment by MFernflower** [comment]
@dmitrij176 the version with the two aryl-chlorine's was - I was proposing swapping the ortho one for hydrogen (i.e monodeschloro pyrifenox)

![1-(para-chlorophenyl)-N-methoxy-2-pyridin-3-ylethanimine](https://user-images.githubusercontent.com/3164942/146288730-6504ed59-4357-411f-9217-064c81630313.png)

**Comment by dmitrij176** [comment]
> @dmitrij176 the version with the two aryl-chlorine's was - I was proposing swapping the ortho one for hydrogen (i.e monodeschloro pyrifenox)
> 
> ![1-(para-chlorophenyl)-N-methoxy-2-pyridin-3-ylethanimine](https://user-images.githubusercontent.com/3164942/146288730-6504ed59-4357-411f-9217-064c81630313.png)

Yes, my bad. 2,4-dichloro on the aromatic ring within the ketoxime scaffold is currently the preferred version.  We already tested DM39-1 which showed a significant decrease in potency and did not pass further than the _in vitro_ phase. Again, if I recall it correctly, it was potent at 100 micromollar, but not at 25.

**Comment by dmitrij176** [comment]
> > @wwjvdsande @dmitrij176 Quick Question - Has pyrifenox itself been tested vs madurella?
> 
> Pyrifenox has been tested in the last biological assay and gave very good results. If I remember correctly, the Minimum Inhibitory Concentration (MIC) was around 0.25 micromolar. Therefore, it will likely be used as a positive control in the future.

**Comment by MFernflower** [comment]
Oh, so modification of the dichloroaryl core isn't on the cards at the moment? DM39-1 was however missing the methylene spacer found in pyrifenox

**Comment by dmitrij176** [comment]
@MFernflower what I think might be useful is to preserve 2 halogens on that aromatic ring. As an option: 2-chloro-4-fluoro.
But potentially multiple options are possible. Testing more analogs will give further insight into the ketoxime series. By now, 5 more molecules are ready for biological evaluation. That batch will investigate the 3 types of substrates:
-extended hydrocarbon chains (as in the original lead molecule, but of various lengths)
-cyclic structures (eg: cyclopropane)
- aromatic ring

**Comment by MFernflower** [comment]
![image](https://user-images.githubusercontent.com/3164942/146300362-7801ee1b-2a69-4329-99bb-0cddc33435b6.png)


@dmitrij176 perhaps something to consider

**Comment by drc007** [comment]
Are all these molecules a mixture of oxime ether isomers? I notice sometimes they are drawn with the oxime ether pointing towards the chlorophenyl ring, sometimes away?
![oximes](https://user-images.githubusercontent.com/7080516/146360710-e14cadac-8349-4a33-8235-3782ab789fc5.png)

**Comment by dmitrij176** [comment]
> Are all these molecules a mixture of oxime ether isomers? I notice sometimes they are drawn with the oxime ether pointing towards the chlorophenyl ring, sometimes away?
> ![oximes](https://user-images.githubusercontent.com/7080516/146360710-e14cadac-8349-4a33-8235-3782ab789fc5.png)

They are a mixture of geometric isomers (E/Z forms). I have corrected the schemes and in all protocols now include the following representation:

<img width="220" alt="image" src="https://user-images.githubusercontent.com/44785501/146403251-d5474daa-1730-4f39-b348-dd8e8e9f1028.png">

Based on what I saw in already synthesised molecules, its always a mixture of E/Z isomers, but with one form being predominant. On TLC, they usually appear as 2 separate spots, which indicates that they can be separated. I am currently looking into that area, but prior to any separations, we need to ensure that they are stable at rt and not interconverting.

With regards to Pyrifenox, having shown good potency results (both  _in vitro_ and _in vivo_), I think it would be interesting to see how isomeric forms would behave in the assay. 

More information on that to follow in the short term.

**Comment by MFernflower** [comment]
@dmitrij176 I opened a repository should you want to put all the ketoxime stuff in there: https://github.com/OpenSourceMycetoma/Series-1B-Ketoximes

**Comment by MFernflower** [comment]
Action item on @dmitrij176 to push information to the new Github repository for the ketoxime fungicides

**Comment by MFernflower** [comment]
@dmitrij176 I had thought up this compound - the SM is super cheap so maybe worth a shot? 


![image](https://user-images.githubusercontent.com/3164942/150254305-b81a1e14-3527-4ba4-b536-06ce48e91f69.png)
